curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The main purpose of the float property in CSS is:
 
 #### --distractors--
 
-Placeholder distractor 1
+(A) To create rounded corners
```

Comment:
```suggestion
To create rounded corners
```
Order of the options are randomized on the quiz page. They should not have any ordering indication in the markdown file. This applies to other distractors and answers.

Reply:
@gikf Ok Thanks ! I'll surely take a look on that and again make a pull request.

Reply:
do not make a new pull request please, update this one

Reply:
Hi @ilenia-magoni  & @gikf !
I've made the required changes.